app/views/maintenances/show.html.haml

```diff
@@ -17,6 +17,10 @@
   %strong= Maintenance.human_attribute_name(:note)
   = @maintenance.note
 
+%p
+  = (link_to image_tag(@maintenance.invoice.norm.url, alt: 'Invoice', class: 'img-thumbnail'), @maintenance.invoice.url ) if @maintenance.invoice?
```

Comment:
Horrible :)
1) if you use 'default_url' correctly you don't need to check the presents of invoice.
2) If you don't use 'default_url' create a decorator which should return a path or nil depends on invoice value and set it as link argument of 'image_tag' helper.
3) Create an maintenances action called for example 'show_invoice'. This page should be redirect you to a blank page with the invoice. Set invoice image in a response container. On bottom add a 'back' button.
```
Responsive imagesi

Images in Bootstrap are made responsive with .img-fluid. max-width: 100%; and height: auto; are applied to the image so that it scales with the parent element.

https://getbootstrap.com/docs/4.0/content/images/#image-thumbnails
```
4) In generally we should avoid logic in views. We using decorators or helpers to manage such issues.
5) Make a thumb format for this views (show/edit/new). The current image is too big